api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/DatimCodeDatasetDefinition.java

```diff
@@ -11,24 +10,40 @@
 import org.openmrs.module.reporting.evaluation.querybuilder.SqlQueryBuilder;
 import org.openmrs.module.reporting.evaluation.service.EvaluationService;
 
+import java.util.List;
+
 /**
  * Simple dataset of location_attribute properties. Actually supported:
  * location_attribute_type.name, location.name and location_attribute.value_reference => Datim Code
  */
 public class DatimCodeDatasetDefinition extends EvaluatableDataSetDefinition {
 
+  protected int typeId;
+
+  protected String columnName;
+
+  protected String getQuery(int typeId, String columnName) {
```

Comment:
are those variable declared protected meant to be used somewhere else in the package? if NOT please declare them as private

Reply:
Yes, they are being used in another file on the same package. Pls see sismaCodeDatasetDefinition.java

Reply:
Ok